content/plugins/luis-guette-moonguard.md

```diff
@@ -0,0 +1,15 @@
+---
+name: MoonGuard
+slug: luis-guette-moonguard
+author_slug: luis-guette
+categories: ['analytics', 'developer-tool']
+description: Plugin that providers utilities to monitor your app status and performance once it is released to production.
+discord_url: https://discord.gg/vCCy4aJxnY
+docs_url: https://docs.moonguard.dev
```

Comment:
Please change this to 'url' instead of 'docs_url' since it is not a markdown file.

Reply:
done!